app/components/find/results/search_result_component.rb

```diff
@@ -9,11 +9,12 @@ class SearchResultComponent < ViewComponent::Base
 
       delegate :age_range_in_years_and_level, :course_length_with_study_mode, to: :course
 
-      def initialize(course:, filtered_by_location: false, has_sites: false)
+      def initialize(course:, filtered_by_location: false, has_sites: false, sites_count: 1)
```

Comment:
```suggestion
      def initialize(course:, filtered_by_location: false, has_sites: false, sites_count: 0)
```

Would this be a better default since `has_sites` defaults to `false`?